css/common.css

```diff
@@ -0,0 +1,83 @@
+@import url("https://fonts.googleapis.com/css2?family=Fjalla+One&family=Outfit:wght@200&display=swap");
+html {
+  scroll-behavior: smooth;
+  overflow-y: scroll;
+  scroll-snap-type: y;
+  scroll-padding: 15vh;
+}
+:root {
+  --red-brown: #a05941;
+  --red-light: #d0b0a6;
+  --green-black: #133032;
+  --green-light: #3e5a5b;
+  --light-grey: #ededee;
+  --white: #fff;
+  --black: #000;
+  --transparent-white: rgba(255, 255, 255, 0.75);
+  --heading-font: "Fjalla One", sans-serif;
+  --body-font: "Outfit", sans-serif;
+}
+* {
+  margin: 0;
+  box-sizing: border-box;
+  border: 0px solid black;
+  font-family: var(--body-font);
+}
+body {
+  background-color: var(--light-grey);
+  padding: 0 2rem;
+}
+.button {
+  padding: 1rem 3rem;
+  background-color: var(--red-brown);
+  text-decoration: none;
+  font-size: 1.1rem;
+  color: var(--white);
+  transition: 0.4s all ease-in-out;
+  cursor: pointer;
+}
+.button:hover {
+  color: var(--black);
+  background-color: var(--red-light);
+}
+.underline {
+  color: var(--red-brown);
+  font-size: 4rem;
+  font-weight: 600;
+  line-height: 0;
+}
+.hero {
+  width: calc(100% + 4rem);
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0 -2rem;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-color: rgba(0, 0, 0, 600);
+  box-shadow: inset 0 0 0 2000px rgba(0, 0, 0, 0.25);
+}
+.hero > * {
+  color: var(--white);
+  fill: var(--white);
+  margin-bottom: 2rem;
+}
+.hero h1 {
+  font-size: 4rem;
+  font-family: var(--heading-font);
+  font-weight: 400;
+  margin-bottom: 2rem;
+}
+@media screen and (min-width: 900px) {
+  main {
+    margin-top: 15vh;
+  }
```

Comment:
Since you're (I think) wanting to set `main` to start where the `header` ends, it would make sense to give it the same margin-top as the height of the header. So if the header is 100px high, you'd give main `margin-top: 100px`. 
To help with keeping track of this, you could even save this value inside a variable, and use it for the header height and the main margin-top. Like this, if you decide to change it, you can update the variable and it will change in both places and still work ✨